.gitignore

```diff
@@ -132,3 +132,5 @@ junit-*
 
 # vscode
 .vscode
+
+.dockerignore
```

Comment:
@danmanor wait, why did you do that?
I don't think we should ignore `.dockerignore` changes in git

Reply:
Why not, I guessed it is mainly for my tests, as I am not sure what `app-sre` or `Prow`  are using to build it.

Reply:
For all cases, regardless of how an image is being built, it doesn't make sense to have ``.dockerignore`` written in `.gitignore`. It's not a generated file or something that you wouldn't want to commit to the repository

Reply:
I understand. The concept of adding my virtual environment name to the git repository seemed odd for me. I will fix it thanks